docs/server/source/production-deployment-template/node-config-map-and-secrets.rst

```diff
@@ -220,10 +220,9 @@ portal using your admin account, click **APIs** and click on **Integration**
 for the relevant API.
 Scroll to the bottom of the page and click the small link
 in the lower right corner, labelled **Download the NGINX Config files**.
-You'll get a ``.zip`` file.
```

Comment:
@ttmc I just confirmed with @muawiakh that the current 3scale portal does not provide a `.zip` file anymore.